packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -156,6 +197,29 @@
   margin-inline-start: theme("margin.auto");
 }
 
+#close,
+#collapse,
+.back-button,
+calcite-action-menu {
```

Comment:
none of these tokens are necessary so we can just remove this style block

Reply:
Could you clarify? These are all set on internally-rendered components that Panel provides, not user-slotted content. I guess we could fallback to the BG color of the Panel (or Header), but this doesn't provide the hover or press states needed.

Reply:
these are covered by button and action tokens.

Reply:
These are internally rendered based on properties (or an auto-magic rendering of `...` menu on behalf of the user): 
![Screenshot 2024-11-06 at 8 14 01 AM](https://github.com/user-attachments/assets/68845849-d044-400f-ba0b-ed5f13b2b250)

The "unthemed" action one above is directly slotted and not affected by these properties - of course in those situations users should theme on the action itself. 

Users should not have to guess or have any knowledge of what components we use internally. It could be an action today - it might be a div or a button tomorrow - that implementation detail should be hidden from the user and they should be able to set these on the component that renders these internals. Having to set the following to affect internally rendered components seems unexpected:

.my-themed-panel {
--calcite-panel-background-color: red;
--calcite-action-background-color: red;
}

 Curious what others think.

Reply:
> Users should not have to guess or have any knowledge of what components we use internally

in this case, yes they should. We should not add a ton of extra tokens when we can just document that `--calcite-action-` tokens are used by putting them in the jsdoc.

Reply:
Following that though, setting “—calcite-action-whatever” on the parent Panel will affect all slotted content anywhere in the component. 

Panels may have dozens of slotted Actions not intended to be styled the same way a user would like to style the internally rendered “clearable” type elements.

So we’d have to doc a public css prop from another component on Panel that would unintentionally affect child components. This feels more confusing and likely frustrating for a user.